Code2Gether-Discord-Bot.Library/BusinessLogic/IBusinessLogic.cs

```diff
@@ -1,9 +1,10 @@
-﻿using Discord;
+﻿using System.Threading.Tasks;
+using Discord;
 
 namespace Code2Gether_Discord_Bot.Library.BusinessLogic
 {
     public interface IBusinessLogic
     {
-        Embed Execute();
+        Task<Embed> Execute();
```

Comment:
Any methods that return a task should have an `Async` suffix. (i.e. `Task<Embed> ExecuteAsync()`)

Reply:
Ah that's right. Can do!